### PR TITLE
Allow temperature field to be defined more dynamically

### PR DESCRIPTION
**Pull request by mneumark**
Allow temperature field to be defined more dynamically allowing for more structures in return JSON

### CODE REVIEW

**Comment by mneumark** [comment]
@metbosch any chance you can take a look at this? I think it would be very helpful to a lot of people.

**Comment by metbosch** [comment]
@mneumark I saw the PR today, sorry for the inconvenience. I did some style fixes and changed one return, could you please take a look?

**Comment by mneumark** [comment]
Yup. Looks good. Thanks for taking a look. 👍🏼

File: index.js [hunk 1]
```diff
@@ -145,18 +145,20 @@ HttpTemperature.prototype = {
    },
 
    getFromObject: function (obj, path, def) {
-      if (!path) return path
+      if (!path) return obj;
```
[the remark this answers is not in the document]
Comment (mneumark) [reply index.js]: Sorry, I really thought I responded. What you did makes more sense. Good thinking.

**Comment by metbosch** [comment]
@mneumark Merged! Let me know if you have any further problem.

**Pull request merged**